project/client/js/inventory.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Created by vova on 29.09.2015.
+ */
+var customers = new Mongo.Collection("customers");
+var problems = [];
+Session.set("prob", "");
+Template.inventory.events({
+
+    "click #cancel": function(){
+        "use strict";
+        location.assign("http://localhost:3000/index.html");
+        location.reload(true);
+    },
+
+    "click .form": function(event){
+        "use strict";
+        if ($(event.target).css("background-color") === "rgb(255, 0, 0)"){
+            $(event.target).css("background-color", "rgb(221, 221, 221)");
+        }
+        else{
+            $(event.target).css("background-color", "rgb(255, 0, 0)");
+        }
+    },
+
+    "click #else": function(){
+        "use strict";
+        var problem = $("#elseInput").val().trim();
+        if (problem) {
+            Session.set("prob", problem);
+        }
+        $("#elseInput").val("");
+    },
+
+    "click #add":function(){
+        "use strict";
+        var bike = $("#bikeName").val().trim();
+        var name = $("#name").val().trim();
+        var phone =$("#phone").val().trim();
+        var deadline = $("#deadline").val().trim();
+        var price = $("#price").val().trim();
+        var troubles = {};
+        $(".form").each(function(){
+            if ($(this).css("background-color") === "rgb(255, 0, 0)"){
+                troubles[$(this).text()] = false;
+            }
+        });
+        problems.forEach(function(trouble){
+            troubles[trouble] = false;
+        });
+        customers.insert({
```

Comment:
there is one thing, which may be bEd: we insert non-validated data into collection. Good point to think about